manager-bundle/src/HttpKernel/ContaoCache.php

```diff
@@ -49,6 +50,15 @@ public function __construct(ContaoKernel $kernel, string $cacheDir = null)
         $this->addSubscriber(new CleanupCacheTagsListener());
     }
 
+    public function getEventDispatcher(): EventDispatcher
+    {
+        if (!$this->eventDispatcher) {
+            $this->eventDispatcher = new EventDispatcher();
+        }
+
+        return $this->eventDispatcher;
```

Comment:
With PHP>=7.4 this could be shortened to:

```suggestion
        return $this->eventDispatcher ??= new EventDispatcher();
```

:upside_down_face: